### PR TITLE
fix links in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by pjfanning**
relates to https://github.com/apache/incubator-pekko-grpc/issues/130

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -207,7 +201,7 @@ Each project must also create and maintain a list of all dependencies and their
 
 Follow these guidelines when creating public commits and writing commit messages.
 
-1. If your work spans multiple local commits (for example; if you do safe point commits while working in a feature branch or work in a branch for a long time doing merges/rebases etc.) then please do not commit it all but rewrite the history by squashing the commits into a single big commit which you write a good commit message for (like discussed in the following sections). For more info read this article: [Git Workflow](http://sandofsky.com/blog/git-workflow.html). Every commit should be able to be used in isolation, cherry picked etc.
+1. If your work spans multiple local commits (for example; if you do safe point commits while working in a feature branch or work in a branch for a long time doing merges/rebases etc.) then please do not commit it all but rewrite the history by squashing the commits into a single big commit which you write a good commit message for (like discussed in the following sections). For more info read this article: [Git Workflow](https://sandofsky.com/blog/git-workflow.html). Every commit should be able to be used in isolation, cherry picked etc.
```
Comment (mdedetrich) [inline CONTRIBUTING.md:204]: Is this valid? I have a feeling this isn't necessarily since squash and merge exists.

**Review by mdedetrich — APPROVED** [review]
lgtm but I think one of the points aren't valid anymore

File: CONTRIBUTING.md (hunk 1 above)
Comment (pjfanning) [reply CONTRIBUTING.md:204]: fair enough - we have this in all our contributing files - I'll remove it here but will need to later make same edit in other repos

**Review by He-Pin — APPROVED** [review]
Lgtm

**Pull request merged**